src/parse/rules/proto_rules.build_defs

```diff
@@ -106,9 +109,14 @@ def proto_library(name, srcs, plugins=None, deps=None, visibility=None, labels=N
         if language == 'go' and not lang_plugins:
             # Go doesn't come by default, so add it here.
             lang_plugins.append('--plugin=protoc-gen-go=' + _plugin(CONFIG.PROTOC_GO_PLUGIN, lang_tools))
+
+        language_out_arguments = ''
+        if language == 'js':
```

Comment:
Minor nit, you could one-line this to `language_out_arguments = 'import_style=commonjs:' if language == 'js' else ''